internal/store/peering_connections.go

```diff
@@ -139,11 +147,13 @@ const maxPerPage = 100
 func (s *Store) AzureContainers(projectID string) ([]atlasv2.CloudProviderContainer, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		result, _, err := s.clientv2.NetworkPeeringApi.ListPeeringContainerByCloudProvider(s.ctx, projectID).
-			PageNum(0).
-			ItemsPerPage(maxPerPage).
-			ProviderName("Azure").
-			Execute()
+		params := &atlasv2.ListPeeringContainerByCloudProviderApiParams{
+			GroupId:      projectID,
+			ProviderName: pointer.Get("Azure"),
+			ItemsPerPage: pointer.Get(maxPerPage),
+			PageNum:      pointer.Get(0),
```

Comment:
can you confirm if this command has pagination and add it if not?

Reply:
we can't because the method has only param projectID, as an example previous method ContainersByProvider has also opts *atlas.ContainersListOptions.    
I would have to modify the signature of the method and add the param.